paper-tabs.html

```diff
@@ -215,28 +207,22 @@
 
       /**
        * If true, the bottom bar to indicate the selected tab will not be shown.
-       *
-       * @attribute nobar
        */
-      nobar: {
+      noBar: {
```

Comment:
I guess my change is a bikeshed too. `no` and `bar` are two words, and I think that the attribute is better expressed in camel case. `noink` is used in a lot of places, so even though I would rather change it, I wasn't sure if I should. What do you think? I'm happy to revert if you think this is too much bike shedding.